test/com/google/enterprise/adaptor/sharepoint/SharePointUserProfileAdaptorTest.java

```diff
@@ -352,18 +355,19 @@ public void testGetDocContent() throws IOException, Exception {
     adaptor.init(new MockAdaptorContext(config, pusher));
 
     ByteArrayOutputStream baos = new ByteArrayOutputStream();
-    GetContentsRequest request = new GetContentsRequest(
+    DocRequest request = new DocRequest(
         new DocId(SharePointUserProfileAdaptor.SOCIAL_ID_PREFIX 
           + "domain\\user1"));
-    GetContentsResponse response = new GetContentsResponse(baos);
+    RecordingResponse response = new RecordingResponse(baos);
     adaptor.getDocContent(request, response);
     
     String responseString = new String(baos.toByteArray(), charset);
     final String golden = "<html><head><title>First &amp; Last"
         + "</title></head><body><h1>First &amp; Last</h1></body></html>";
     assertEquals(golden, responseString);
 
-    assertFalse(response.isNotFound());
+    String responseState = response.getState().toString();
```

Comment:
Why not just assertFalse(response.getState().equals(State.NOT_FOUND))? Or even better, using the Hamcrest matchers:

import static org.hamcrest.CoreMatchers.equalTo;
import static org.hamcrest.CoreMatchers.not;

assertThat(response.getState(), not(equalTo(State.NOT_FOUND)));

Reply:
Done. Thanks